ktor-server/ktor-server-core/jvm/src/io/ktor/routing/RouteSelector.kt

```diff
@@ -265,7 +265,7 @@ internal object TrailingSlashRouteSelector : RouteSelector(RouteSelectorEvaluati
         else -> RouteSelectorEvaluation.Failed
     }
 
-    override fun toString(): String = "/"
+    override fun toString(): String = "<trailing_slash>"
```

Comment:
Not sure about it. What if someone already has a custom 'Route.toString' implementation based on RouteSelector.toString implementation? It may accidentally stop working

Reply:
ok, reverted